torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -789,6 +789,40 @@ RegisterOperators reg({
             };
           }
         }),
+    Operator(
+      prim::DictConstruct,
```

Comment:
You will need to add a case for prim::DictConstruct & prim::DictIndex in alias_analysis